src/base/orgs/TransferOwnership.svelte

```diff
@@ -58,15 +68,15 @@
   };
 </script>
 
-{#if state === State.Success}
+{#if state === State.Success && newOwner}
   <Modal floating small>
     <div slot="title">
       ✅
     </div>
 
     <div slot="subtitle">
-      The ownership of <strong>{formatAddress(org.address)}</strong> was
-      successfully transfered to <strong>{newOwner}</strong>.
+      The ownership of <Address address={org.address} {config} resolve noAvatar noBadge compact /> was
+      successfully transfered to <Address address={newOwner} {config} resolve compact noAvatar noBadge />.
```

Comment:
I would prefer to leave these as they were for now. It exhibits strange behavior sometimes and when it fails to load there is just a gap in the text.